### PR TITLE
Update documentation to account for addition of new sideloading commands to Yo Office (see https://github.com/OfficeDev/generator-office/commit/f95809fc1c2b8490cbedb3827d0d1754ad3d2238)

### PR DESCRIPTION
**Pull request by TCourtneyOwen**
-Add new help topic for sideloading add-ins via "npm run sideload"
- Link to new help topic off existing create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/sideload-office-addin-using-sideload-command.md](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/testing/sideload-office-addin-using-sideload-command.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201807241734535502-263%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=AbstzHU9jmJCTVMhkE3lqp1%2FlW0yhI%2BNcudZ0pYpy%2F0%3D&st=2018-07-24T17%3A31%3A05Z&se=2018-08-24T17%3A36%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/sideload-office-addin-using-sideload-command.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (AlexJerabek) [inline docs/testing/sideload-office-addin-using-sideload-command.md]: Please update the ToC to include this file.

File: docs/testing/sideload-office-addin-using-sideload-command.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+title: Sideload Office Add-ins Using Sideload Command
+description: ''
+ms.date: 07/24/2018
+---
+
+# Sideload Office Add-ins for testing using the **sideload command**
+ (**NOTE**: this method only works for Excel, Word and PowerPoint add-ins).
```
Comment (AlexJerabek) [inline docs/testing/sideload-office-addin-using-sideload-command.md]: Same [markdown note](https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master) comment as before.

File: docs/testing/sideload-office-addin-using-sideload-command.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Sideload Office Add-ins Using Sideload Command
```
Comment (AlexJerabek) [inline docs/testing/sideload-office-addin-using-sideload-command.md]: Titles use lowercase for non-proper nouns ("Sideload Office Add-ins using the sideload command")

File: docs/testing/sideload-office-addin-using-sideload-command.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+description: ''
+ms.date: 07/24/2018
+---
+
+# Sideload Office Add-ins for testing using the **sideload command**
+ (**NOTE**: this method only works for Excel, Word and PowerPoint add-ins).
+
+1. Open a command prompt as an administrator, and change directories to the root of your add-in project folder.
```
Comment (AlexJerabek) [inline docs/testing/sideload-office-addin-using-sideload-command.md]: Remove the comma (or split this into two steps).

File: docs/testing/sideload-office-addin-using-sideload-command.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+2. Run the following command to start a local web server instance on port 3000 to serve up your add-in project: "**npm run start**"
+
+3. Open a second command prompt as an administrator, and change directories to the root of your add-in project folder.
```
Comment (AlexJerabek) [inline docs/testing/sideload-office-addin-using-sideload-command.md]: Same comma comment

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 6]
```diff
@@ -6,14 +6,14 @@ ms.date: 01/25/2018
 
 # Sideload Office Add-ins for testing
 
-You can install an Office Add-in for testing in an Office client running on Windows by using a shared folder catalog to publish the manifest to a network file share. 
+You can install an Office Add-in for testing in an Office client running on Windows by either 1) using a shared folder catalog to publish the manifest to a network file share, or 2) [running the "**npm run sideload**" command from the root of the add-in project folder (**NOTE**: this method only works for Excel, Word and PowerPoint add-ins)](sideload-office-addin-using-sideload-command.md).
```
Comment (AlexJerabek) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]: Use the oxford comma ("Excel, Word, and Powerpoint").

**Comment by TCourtneyOwen** [comment]
Addressed all comments and submitting new iteration

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/sideload-office-addin-using-sideload-command.md](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/testing/sideload-office-addin-using-sideload-command.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201807261607361504-263%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qyfProXsyu4KiIJ4cbT7dnAx%2Fad4%2B2F9LBaMgYetXC8%3D&st=2018-07-26T16%3A03%3A46Z&se=2018-08-26T16%3A08%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 7]
```diff
@@ -6,7 +6,10 @@ ms.date: 01/25/2018
 
 # Sideload Office Add-ins for testing
 
-You can install an Office Add-in for testing in an Office client running on Windows by either 1) using a shared folder catalog to publish the manifest to a network file share, or 2) [running the "**npm run sideload**" command from the root of the add-in project folder (**NOTE**: this method only works for Excel, Word and PowerPoint add-ins)](sideload-office-addin-using-sideload-command.md).
+You can install an Office Add-in for testing in an Office client running on Windows by one of the following methods:
+
+- using a shared folder catalog to publish the manifest to a network file share (instructions below)
```
Comment (AlexJerabek) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]: Please capitalize the first word of each bulleted list entry.

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 8]
```diff
@@ -6,7 +6,10 @@ ms.date: 01/25/2018
 
 # Sideload Office Add-ins for testing
 
-You can install an Office Add-in for testing in an Office client running on Windows by either 1) using a shared folder catalog to publish the manifest to a network file share, or 2) [running the "**npm run sideload**" command from the root of the add-in project folder (**NOTE**: this method only works for Excel, Word and PowerPoint add-ins)](sideload-office-addin-using-sideload-command.md).
+You can install an Office Add-in for testing in an Office client running on Windows by one of the following methods:
+
+- using a shared folder catalog to publish the manifest to a network file share (instructions below)
+- [running the "**npm run sideload**" command from the root of the add-in project folder.](sideload-office-addin-using-sideload-command.md) [!NOTE] this method only works for Excel, Word, and PowerPoint add-ins).
```
Comment (AlexJerabek) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]: I believe Markdown notes have to be on their own lines with '>' preceding the [!NOTE] and text. Here's the example from the docs site.
```markdown
> [!NOTE]
> Information the user should notice even if skimming.
```

Comment (AlexJerabek) [reply docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]: "This" at the start of the note should also be capitalized.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/sideload-office-addin-using-sideload-command.md](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/testing/sideload-office-addin-using-sideload-command.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/TCourtneyOwen/office-js-docs-pr/blob/add_commandLine_sideloading_step/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201807261859306207-263%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oDzNTuugxDCJKp5nPrHaSQ%2B36650%2FJ5K3jGWCZzweCQ%3D&st=2018-07-26T18%3A57%3A08Z&se=2018-08-26T19%3A02%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**